packages/components/src/notice/index.js

```diff
@@ -7,17 +7,19 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
+import Dashicon from '../dashicon';
 import { __ } from '@wordpress/i18n';
 
 function Notice( { className, status, children, onRemove = noop, isDismissible = true } ) {
-	const classNames = classnames( className, 'notice notice-alt notice-' + status, {
+	const classNames = classnames( className, 'components-notice components-notice--' + status, {
```

Comment:
Should this be `__` instead of `--`

Reply:
Actually, it's looks like this is a modifier. I think in general our guideline is to use unprefixed booleans. So maybe just: `is-warning`, `is-error`...

Reply:
@jasmussen will know what's the best way to deal with it 😃

Reply:
I think I'm feeling what Riad is saying. 

I believe we have these notices:

- Green (Success)
- Orange (Warning)
- Red (Error)

I can't recall if there's also a blue informational notice, if there isn't there probably should be.

The default notice should be an informational notice. In addition to that, we could have is-succes, is-warning, is-error. 

Did this make sense?

Reply:
Yep. It's consistent with what we do in other components. I have just committed the change using these new modifiers.